tests/README.txt

```diff
@@ -0,0 +1,17 @@
+
+The xross/feat/usb_shim branches need pulling into the mainlines of the
+folliwing repos
+
+xsim 
+xsimutils
+tools_xmostest
```

Comment:
Is this true? (Also, following is spelt wrongly ;-))

Reply:
Fixed this.